src/main/java/com/egen/AppInitialize.java

```diff
@@ -20,6 +19,6 @@
 
 	@Override
 	protected String[] getServletMappings() {
-		return new  String[]{};
+		return new  String[]{"/"};
```

Comment:
its a good practice to have your context path prefixed with /api

Reply:
Thanks for mentioning it, Arif.
Sure will follow this practice in the future.